pyrevolve/genotype/neat_brain_genome/crossover.py

```diff
@@ -1,24 +1,32 @@
+import random
+
 class NEATCrossoverConf:
     def __init__(self):
         self.mate_average = True
         self.interspecies_crossover = True
 
 
-def standard_crossover(parent_individuals, genotype_conf, crossover_conf):
+def standard_crossover(parents, NeatCrossoverConf, crossover_conf, lsystem_conf):
     """
     Creates an child (individual) through crossover with two parents
```

Comment:
my fault, this should be "creates a child (genotype) through crossover with two parents"